src/sqs-redriver.js

```diff
@@ -0,0 +1,125 @@
+import { validateSchema, joi } from '@springworks/input-validator';
+import sqs_instance from './sqs-instance';
+
+const deserialized_queue_params_schema = joi.object().required().keys({
+  source_queue_url: joi.string().required(),
+  target_queue_url: joi.string().required(),
+});
+
+const params_schema = joi.object().required().keys({
+  sns_event: joi.object().required().keys({
+    Records: joi.array().required().items(joi.object().required().keys({
+      EventVersion: joi.string().optional(),
+      EventSubscriptionArn: joi.string().optional(),
+      EventSource: joi.string().optional(),
+      Sns: joi.object().required().keys({
+        SignatureVersion: joi.string().optional(),
+        Timestamp: joi.string().optional(),
+        Signature: joi.string().optional(),
+        SigningCertUrl: joi.string().optional(),
+        MessageId: joi.string().optional(),
+        Message: joi.string().required().notes('Expected to be stringified JSON, see deserialized_message_schema'),
+        MessageAttributes: joi.object().optional(),
+        Type: joi.string().optional(),
+        UnsubscribeUrl: joi.string().optional(),
+        TopicArn: joi.string().optional(),
+        Subject: joi.string().optional().allow(''),
```

Comment:
To make it more clear what the SNS event actually contains (it's almost impossible from their docs).